modules/packages/DynamicLoading.chpl

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2020-2025 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Can we make tests out of the code blocks? We'd need to be able to
+// add a 'precomp' and 'compopts'.
+//
+
+/*
+  Support for dynamic loading in Chapel.
+
+  .. warning::
+
+    This module is unstable. Support for dynamic loading is experimental
+    and its representation could change at any time.
```

Comment:
This is not needed and auto-added by chpldoc. I think this text would do well in `@unstable`